src/sidebar/services/toast-messenger.ts

```diff
@@ -30,9 +46,25 @@ type MessageOptions = {
 // @inject
 export class ToastMessengerService {
   private _store: SidebarStore;
+  private _window: Window;
+
+  /**
+   * This just holds a set of delayed messages that need to be published as soon
+   * as the window gets focused
+   */
+  private _delayedMessagesQueue: Set<MessageData>;
```

Comment:
Do you think so? 🤔. To me it sounds more natural in plural, as it is a "queue containing delayed messages".

Reply:
Ok, never mind. I think I actually get your point here.

The plural would make more sense if it was called just `delayedMessages`, but by naming it queue, then `delayedMessageQueue` is more logical, like in `messageRepository`, `messageStore`, etc.

Reply:
> Do you think so? 🤔

Yes. As a point of comparison, grep for similar variables in the code:

```sh
ag '[a-z](Set|List|Queue)\b' src/
```

This turns up `BucketSet`, `ThreadList` etc.

I'm not certain of the proper explanation, but I think it would be something like this: "message" is referring to the kind of object in the "queue", rather than the number.